test/test_unit.jl

```diff
@@ -1224,6 +1224,25 @@ end
               flux(u, normal_direction, equations)
     end
 
+    # check consistency between 1D and 2D HLLC fluxes
+    u_1d = SVector(1.1, -0.5, 5.5)
+    u_2d = SVector(u_1d[1], u_1d[2], 0.0, u_1d[3])
+    normal_1d = SVector(-1.0)
+    normal_2d = SVector(normal_1d[1], 0.0)
```

Comment:
Based on what Hendrik wrote: It probably makes sense to test the equivalence of the 1D flux for a non-unit length normal vector (like `normal_1d = SVector(-2.0)`) with the exisiting to 2D implementation.

Reply:
Will do - thanks for the suggestion.